spinalcordtoolbox/scripts/sct_label_vertebrae.py

```diff
@@ -418,13 +427,25 @@ def main(args=None):
              verbose=verbose,
              is_sct_binary=True,
              )
-    # Clean labeled segmentation
-    printv('\nClean labeled segmentation (correct interpolation errors)...', verbose)
-    clean_labeled_segmentation('segmentation_labeled.nii', 'segmentation.nii', 'segmentation_labeled.nii')
+
+    if clean_labels:
+        # Clean labeled segmentation
+        printv('\nClean labeled segmentation (correct interpolation errors)...', verbose)
+        clean_labeled_segmentation('segmentation_labeled.nii', 'segmentation.nii', 'segmentation_labeled.nii')
 
     # label discs
     printv('\nLabel discs...', verbose)
-    label_discs('segmentation_labeled.nii', verbose=verbose)
```

Comment:
isn't `label_discs` not used at all anymore?

Reply:
No, it is used in `spinalcordtoolbox/vertebrae/core.py` to create the image with the label (in the straighten space).